src/living/helpers/gui2d/control.ts

```diff
@@ -389,6 +404,11 @@ export class Control2D {
       }
     }
 
+    /**
+     * Render the transform.
```

Comment:
This comment is incorrect, what this function does is just changing the transform matrix for the current rendering context IMO.